pymatgen/io/qchem/inputs.py

```diff
@@ -52,16 +55,62 @@ def __init__(self, molecule, rem, opt=None, pcm=None, solvent=None, smx=None):
         self.solvent = lower_and_check_unique(solvent)
         self.smx = lower_and_check_unique(smx)
 
-        # Make sure molecule is valid: either the string "read" or a pymatgen molecule object
+        # Make sure molecule is valid
 
         if isinstance(self.molecule, str):
             self.molecule = self.molecule.lower()
             if self.molecule != "read":
                 raise ValueError(
                     'The only acceptable text value for molecule is "read"')
+
+        # Allows for multiple molecules, which is necessary for fsm jobs
+        elif isinstance(self.molecule, dict):
```

Comment:
Shyue might lose his mind with all this type checking despite the fact that we were doing some already. Not sure I see a cleaner way, though...

Reply:
The alternatives that I see are try-except and checking by attribute (which, while Pythonic, isn't as safe).